docs/api/non_geo_datasets.csv

```diff
@@ -36,6 +36,7 @@ Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
 `Tropical Cyclone`_,R,GOES 8--16,"108,110",-,256x256,4K--8K,MSI
 `UC Merced`_,C,USGS National Map,"2,100",21,256x256,0.3,RGB
 `USAVars`_,R,NAIP Aerial,100K,-,-,4,"RGB, NIR"
+`USAVarsFeaturesExtracted`_,R,NAIP Aerial,100K,-,-,4,"RGB, NIR"
```

Comment:
Same here, no need for a new row, the values are the same and the link will link to the above sections